src/ray/protobuf/autoscaler.proto

```diff
@@ -317,11 +317,32 @@ message DrainNodeReply {
   string rejection_reason_message = 2;
 }
 
+message NodeGroupConfig {
+  map<string, double> resources = 1;
+  optional uint64 max_num_nodes = 2;
```

Comment:
Count as
```
    // The minimum number of instances to launch.
    int32 min_count = 3;
    // The maximum number of instances to launch.
    int32 max_count = 4;
```

Reply:
Can it be negative?

Reply:
-1 for max means infinite. Let's sync offline first and decide how to move forward.